### DISCUSSION TITLE
How to reverse the display of files from largest to smallest to smallest to largest?

### DISCUSSION THREAD

**Discussion by Dialga** [Byron/dua-cli#109] [Q&A]
Sometimes you have a lot of small files in a directory, it would be helpful to reverse the output to show the smallest files at the bottom when ran with a -r flag.

**Comment by Byron** [comment]
Pressing `s` should do exactly that.

<img width="1133" alt="Screenshot 2021-10-01 at 22 17 26" src="https://user-images.githubusercontent.com/63622/135635401-b8575d86-f5d0-43d8-96b0-1ed8c9e9d633.png">

<img width="1084" alt="Screenshot 2021-10-01 at 22 20 12" src="https://user-images.githubusercontent.com/63622/135635861-9ee9c449-2a10-412d-ad2c-cd87e1de2193.png">

**Comment by Dialga** [comment]
Yes but I meant for non interactive use, is there way I can pass s to the cli?

**Reply by Byron** [reply]
Indeed, there is no built-in flag for this yet, a PR is certainly welcome.

Right now the easiest way to achieve this might be `dua a --no-total | tac`.
